src/main/java/de/samply/share/broker/rest/Searchbroker.java

```diff
@@ -238,6 +240,70 @@ public Response getAllDirectoryIdOptions() {
     }
   }
 
+  /**
+   * Save the selected biobanks for the negotiator.
+   * @return <CODE>200</CODE> the selected biobanks are saved
+   *        <CODE>400</CODE> if the inquiry id can't be parsed
+   *        <CODE>401</CODE> if the provided credentials do not allow to read this inquiry
+   *        <CODE>404</CODE> if no inquiry with this id was found
+   *        <CODE>500</CODE> on any other error
+   */
+  @Secured
+  @Path("/selectedBiobank")
+  @POST
+  @APIResponses({
+          @APIResponse(
+                  responseCode = "200",
+                  description = "ok"),
+          @APIResponse(responseCode = "400", description = "Query by n-Token not found"),
+          @APIResponse(responseCode = "500", description = "Internal Server Error")
+  })
+  @Operation(summary = "Save the selected biobanks for the negotiator")
+  public Response saveSelectedBiobank(@QueryParam("ntoken") String ntoken,
+                                      @QueryParam("selectedBiobanks") String selectedBiobanks) {
+    try {
+      new NTokenHandler().saveSelectedBiobank(ntoken, selectedBiobanks);
+      return Response.ok().build();
+    } catch (NotFoundException e) {
+      LOGGER.error(e.getMessage());
+      return Response.status(Response.Status.BAD_REQUEST.getStatusCode(), e.getMessage()).build();
+    } catch (Exception e) {
+      LOGGER.error(e.getMessage());
+      return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
+    }
+  }
+
+  /**
+   * Save the selected biobanks for the negotiator.
+   * @return <CODE>200</CODE> the selected biobanks are saved
+   *        <CODE>400</CODE> if the inquiry id can't be parsed
+   *        <CODE>401</CODE> if the provided credentials do not allow to read this inquiry
+   *        <CODE>404</CODE> if no inquiry with this id was found
+   *        <CODE>500</CODE> on any other error
+   */
+  @Secured
+  @Path("/selectedBiobank")
+  @GET
+  @APIResponses({
+          @APIResponse(
+                  responseCode = "200",
+                  description = "ok"),
+          @APIResponse(responseCode = "400", description = "Query by n-Token not found"),
+          @APIResponse(responseCode = "500", description = "Internal Server Error")
+  })
+  @Operation(summary = "Save the selected biobanks for the negotiator")
+  public Response getSelectedBiobank(@QueryParam("ntoken") String ntoken) {
+    try {
+      return Response.ok(new NTokenHandler().getSelectedBiobank(ntoken)).build();
+    } catch (SQLException e) {
+      LOGGER.error(e.getMessage());
+      return Response.status(Response.Status.BAD_REQUEST.getStatusCode(), e.getMessage()).build();
+    } catch (Exception e) {
+      LOGGER.error(e.getMessage());
+      return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
+    }
+  }
+
```

Comment:
Suggestion: renaming /selectedBiobank to /saveSelectedBiobanks and /getSelectedBiobanks